example/bench_usage/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.14)
+project(dwarf_bench_test)
+
+find_package(TBB REQUIRED)
+find_package(Boost COMPONENTS program_options filesystem REQUIRED)
+find_package(OpenCL REQUIRED)
+find_package(oclhelpers 0.1.3 REQUIRED)
+find_package(oneDPL REQUIRED)
+find_package(dbench REQUIRED)
+
+add_executable(bench_usage main.cpp)
+target_link_libraries(bench_usage dbench::dbench intlc)
```

Comment:
Why intlc?

Reply:
Forgot to ask :)
I had `undefined reference` on `_intel_fast_memset` in `libstandalone_scan.a`
How should we handle this?

Reply:
I don't think that is a real dependency that we need to resolve, but rather some local misconfiguration.

Reply:
Maybe try building the example separately with some gcc?

Reply:
Still can't build example without `intlc`
Getting 
```
/usr/bin/ld: /usr/local/lib/libdbench.a(bench.cpp.o): undefined reference to symbol '_intel_fast_memcpy'
/usr/bin/ld: /opt/intel/oneapi/compiler/2022.2.1/linux/compiler/lib/intel64_lin/libintlc.so.5: error adding symbols: DSO missing from command line
```
in the Docker image. In the OmniSci Conda env even more linker errors on the same function
Maybe we can somehow take into account `intlc` in `dbench` CMake?

Reply:
I was able to reproduce

Reply:
So the reason for this is that the compiler in the base kit is built from closed source and uses intel-specific extensions. I assume it should 'just work' with an opensource compiler. There is a way to get containers with a ready-to-use compiler (https://github.com/intel/llvm/pkgs/container/llvm), but the current latest version doesn't work with the latest oneDPL. Let's not include anything additional to the dbench interface and build it with opensource components only.